REF_LICENSE

```diff
@@ -0,0 +1,11 @@
+--2021-03-01 11:53:03--  https://raw.githubusercontent.com/qiime2/q2lint/master/q2lint/REF_LICENSE
```

Comment:
This file (`REF_LICENSE`) looks like it got committed on accident - any idea where this came from? Either way, please remove this (in a new commit) - thanks!